README.md

```diff
@@ -254,11 +256,13 @@ python3 tools/make_transforms_docs.py check README.md
 Alternatively, you can start a web server that rebuilds the documentation
 automatically when a change is detected by running `make livehtml`
 
-## Hall of fame
+## Competitions won with the library
 Albumentations are widely used in Computer Vision Competitions at Kaggle an other platforms.
 
 You can find their names and links to the solutions [here](docs/hall_of_fame.rst).
 
+## Used by
+<img src="https://habrastorage.org/webt/ce/bs/sa/cebssajf_5asst5yshmyykqjhcg.png" width="100"/>
```

Comment:
Maybe we should also add a clickable link to the Lyft site? Something like `<a href="https://www.lyft.com/"><img src="https://habrastorage.org/webt/ce/bs/sa/cebssajf_5asst5yshmyykqjhcg.png" width="100"/></a>`

Reply:
Good idea. Fixed.